### PR TITLE
Add keyboard incognito mode #3855

### PR DESCRIPTION
**Pull request by michaelsam94**
-add incoginto mode in prefrences
-apply incoginto mode on subject in compose_message activity

Please ensure that your pull request meets the following requirements - thanks !

* Follows our existing [codestyle](https://github.com/k9mail/k-9/wiki/CodeStyle).
* Does not break any unit tests.
* Contains a reference to the issue that it fixes.
* For cosmetic changes add one or multiple images, if possible.

### CODE REVIEW

**Comment by wiktor-k** [comment]
Hi, thanks for your PR!

Just out of curiosity what's the use case? Why would one use incognito for writing just e-mails and not the rest of the system?

**Comment by michaelsam94** [comment]
may be the email itself like browser incoginto mode for all browser

File: app/core/src/main/java/com/fsck/k9/K9.java [hunk 1]
```diff
@@ -176,6 +176,7 @@
     private static boolean wrapFolderNames = false;
     private static boolean hideUserAgent = false;
     private static boolean hideTimeZone = false;
+    private static boolean enableIncoginto = false;
```
[the remark this answers is not in the document]
Comment (michaelsam94) [reply app/core/src/main/java/com/fsck/k9/K9.java]: Can you specify editText in app need to be incognito mode I made subject only to showcase but you can use it anywhere in application I see the K9Edittext will be reusable just replace edit text with custom edittext and also this solution will be used anywhere in application and the value of incognito will be based on user prefs

**Comment by cketti** [comment]
Thanks for this proof of concept. But it looks like nobody is willing to put in the work to make this available in all of the app.
And I currently don't see what a test would look like to make sure we don't accidentally introduce EditTexts that don't support incognito mode in the future. So for now we'll pass on this feature.

**Pull request closed without merging**